src/h5cpp/dataspace/type_trait.hpp

```diff
@@ -55,6 +58,11 @@ class TypeTrait {
     return Scalar();
   }
 
+  const static DataspaceType & get(const T &, DataspacePool &) {
+    const static DataspaceType & cref_ = Scalar();
+    return cref_;
+  }
```

Comment:
DataspacePool is not used in the function, why it's here?

Reply:
It is becuase to have the common interface with other `get()` methods of the factory. The `pool` is used for `Simple` datatypes e.g. with `vector<T>` Type.